openedx/core/djangoapps/user_api/accounts/__init__.py

```diff
@@ -83,7 +83,7 @@
 # into the required field or enter it incorrectly.
 REQUIRED_FIELD_NAME_MSG = _(u"Please enter your Full Name.")
 REQUIRED_FIELD_CONFIRM_EMAIL_MSG = _(u"The email addresses do not match.")
-REQUIRED_FIELD_COUNTRY_MSG = _(u"Please select your Country.")
+REQUIRED_FIELD_COUNTRY_MSG = _(u"Please select your country of full-time residence.")
```

Comment:
@Ayub-Khan Could we please make some edits to the existing messages too, for consistency? The nouns/field names should not be capitalized, and it's preferred not to say "Please" even though it seems more polite:
"Enter your full name." 
"Select your country or region of residence." 
Not sure whether "City" is used? "Enter your city."
"Tell us your goals."
"Select the highest level of education you have completed."